tmt/steps/provision/bootc.py

```diff
@@ -271,6 +283,8 @@ def _build_bootc_disk(self, containerimage: str, image_builder: str) -> None:
             "build",
             "--type",
             "qcow2",
+            "--rootfs",
+            rootfs,
```

Comment:
It would be better if add condition for `fedora-bootc` as `centos-bootc` does not need it.

Reply:
Hi @HuijingHei ,

The `centos-bootc` has default rootfs configured in container image https://gitlab.com/redhat/centos-stream/containers/bootc/-/blob/c9s/centos-stream-common.yaml?ref_type=heads#L17. But `fedora-bootc` does not.

The `--rootfs` in bib will overwrite rootfs configured in container image. https://github.com/osbuild/bootc-image-builder/blob/d42ad7b6d06cb9cffff36a7ccd6754c337303dfd/bib/cmd/bootc-image-builder/main.go#L256. So do not need additional code for that and it's not the checking point of this plugin.

Reply:
Makes sense to me, thanks!